metadetect_driver/MetaDetectRunner.py

```diff
@@ -1,35 +1,47 @@
+import importlib.metadata
+import logging
+import warnings
 from copy import deepcopy
 from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
 
-import numpy as np
-import pandas as pd
 import galsim
-import ngmix
-import metadetect
 import galsim.roman as roman
+import metadetect
+import ngmix
+import numpy as np
+import pyarrow as pa
+import pyarrow.parquet as pq
 import sep
+import yaml
 from astropy import wcs
-import os, sys
 
 from pyimcom.analysis import OutImage, Mosaic
-from pyimcom.config import Settings as Stn
+from pyimcom.config import Settings
+
+from .config import parse_driver_config
+
+
+logger = logging.getLogger(__name__)
 
-from .config import parse_driver_cfg
-import warnings
-import yaml
 
-# load default metadetect config file
-with open('../config/metadetect_default.yaml', 'r') as file:
-        METADETECT_CONFIG= yaml.safe_load(file)
+_DEFAULT_METADETECT_CONFIG = Path(__file__).parent.parent / "config" / "metadetect_default.yaml"
 
 
+def load_default_metadetect_config():
+    with open(_DEFAULT_METADETECT_CONFIG, "r") as file:
+        config = yaml.safe_load(file)
+
+    return config
+
 
 class MetaDetectRunner:
     """
     Class to run MetaDetection on PyIMCOM coadds (Mosaic or OutImage objects).
     Stores the input coadds, MetaDetection config, and driver config, and provides
     methods to build catalogs from the multi-band imaging.
     """
+
     NATIVE_PIX = 0.11  # arcsec/pixel (Roman native pixel scale)
```

Comment:
This is also provided through `galsim.roman.pixel_scale`. Is that the best universal source for this? Would be better to not have this hardcoded in multiple places...